src/product.njk

```diff
@@ -105,5 +105,12 @@ heroLg: true
             </div>
             <img src="../images/node-red-hexagon.png" />
         </div>
+	<div class="product-left">
+            <div>
+                <h2>Edge Device Management</h2>
+                <p>Deploy flows to edge devices with the click of a button. Updating flows in a few megabytes, no need to update Docker images, no need to compromise your security on your devices.</p>
+            </div>
+            <img src="../images/pictograms/community_red.png" />
```

Comment:
Should these pictograms be moved to the components?

Reply:
@ZJvandeWeg quite possibly, although the website isn't able to utilise forge-ui-components at the moment as we are using `njk` and not `vue`. We utilise the CSS classes that `forge-ui-components` offers, but nothing more at the moment (in the website)